deploy/manifests/sample-deployments/kong-enterprise-hybrid/kong-enterprise-services.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kong-proxy
+  namespace: kong
+spec:
+  type: NodePort
+  ports:
+    - name: proxy
+      nodePort: 30800
+      port: 80
+      targetPort: 8000
+      protocol: TCP
+    - name: proxy-ssl
+      nodePort: 30843
+      port: 443
+      targetPort: 8443
+      protocol: TCP
```

Comment:
Choosing specific NodePort ports here and elsewhere seems risky. Ideally we don't want to include example config that may conflict with other Services in user environments, even if that means it's not possible to provide other examples that rely on a known port.

This occurs in several places throughout the PR, and should be addressed in all, but I'm only commenting on this particular instance.

Reply:
This is more an example about "changing" the default behaviour of the `LoadBalancer` services, for example in an on-premise environment where we cannot use Kubernetes provided LoadBalancer and we need to attach some external load balancer to our Kong cluster we usually use NodePorts. We can remove these "architectural" changes/choices from the sample deployment folders